dev/reference/configuration/tidb-server/configuration.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: TiDB 配置参数
 category: reference
+aliases: ['/docs-cn/op-guide/configuration/','/docs-cn/sql/server-command-option/','/docs-cn/sql/tidb-server/']
```

Comment:
I see you added three aliases. Will the content in those three files be moved here?

Reply:
I think the latter two are for redirection purposes. The files are not actually in TOC. Morgan did this in docs. But maybe we need to look at each file later after renaming.

Reply:
👌